ios/src/TangramMap/TGLabelPickResult.h

```diff
@@ -0,0 +1,43 @@
+//
+//  TGLabelPickResult.h
+//  tangram
+//
+//  Created by Karim Naaji on 11/02/16.
+//
+
+#import <Foundation/Foundation.h>
+#import "TGGeoPoint.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ A label type used to differentiate icon and text labels when selecting them on a map view
+ */
+typedef NS_ENUM(NSInteger, TGLabelType) {
+    /// A type for icon labels (sprites)
+    TGLabelTypeIcon = 0,
+    /// A type for text labels
+    TGLabelTypeText,
+};
+
+/**
+ Data structure holding the result of a label selection that occured on the map view.
+
+ See `-[TGMapViewController pickLabelAt:]` and `[TGMapViewDelegate mapView:didSelectLabel:atScreenPosition:]`.
+ */
+@interface TGLabelPickResult : NSObject
+
+- (instancetype) initWithCoordinates:(TGGeoPoint)coordinates type:(TGLabelType)type properties:(NSDictionary *)properties;
```

Comment:
I've set `skip_undocumented` to `YES` in the jazzy yaml description so we don't expose things that the user shouldn't use, here it is an initializer that should only be used by `TGMapViewController` to create the results of a selection query but I can still document it and put a note about that?